build_tools/cmake/iree_macros.cmake

```diff
@@ -654,7 +654,7 @@ function(iree_validate_required_arguments
   endforeach()
 endfunction()
 
-# iree_compile_flags_for_patform
+# iree_compile_flags_for_platform
```

Comment:
Thanks, but I'm hesitant to merge this single comment change on its own. Each commit triggers a CI run that has some nontrivial costs, so I'd be weighing the utility of this fix against that. Are there other typo fixes or structural changes that could be batched with this PR?

Reply:
It's okay. I understand. I will fix this typo in another PR to reduce the costs of running CI.